masonite/drivers/BaseUploadDriver.py

```diff
@@ -0,0 +1,26 @@
+import os
+from masonite.exceptions import ExtensionException
+
+
+class BaseUploadDriver(object):
+    """
+    Class base for upload drivers
+    """
+    def __init__(self):
+        # this will accept all file types
+        self.accept_file_types = None
+
+    def accept(self, *args, **kwargs):
+        """
+        Set files to accept
+        """
+        self.accept_file_types = args
+        return self
+
+    def valid_extension(self, filename):
```

Comment:
let's change this to `validate_extension` to add more of an "action" word.

Reply:
Ok, it's true.